astropy/time/tests/test_basic.py

```diff
@@ -313,6 +313,7 @@ def test_location_array(self):
         assert t5.location.shape == t5.shape
         assert t5.tdb.shape == t5.shape
 
+    @pytest.mark.remote_data
```

Comment:
@taldcroft - the test itself here should work fine with the IERS B file we ship. I forgot, but is there a way to turn off explicitly the auto-download feature so that we don't get a warning?

Reply:
Great! So, we just have to be sure we re-enable it at the end of the test, otherwise it will affect everything else...

Reply:
Probably want to set with the config context manager.

Reply:
Done.